src/controllers/module/event/eventController.ts

```diff
@@ -424,7 +451,48 @@ export class EventController {
 
 	public getEventOverview() {
 		return async (req: Request, res: Response) => {
-			res.render('page/course/module/events/events-overview')
+			const event = res.locals.event
+			const eventDateWithMonthAsText: string = DateTime.convertDate(event.dateRanges[0].date)
+			res.render('page/course/module/events/events-overview', {eventDateWithMonthAsText})
+		}
+	}
+
+	public getAttendeeDetails() {
+		return async (req: Request, res: Response) => {
+			const bookings = res.locals.bookings
+			const bookingId = req.params.bookingId
+			const booking = bookings.find(function(booking: Booking) {
+				return booking.id == bookingId
+			})
+
+			const event = res.locals.event
+			const eventDateWithMonthAsText: string = DateTime.convertDate(event.dateRanges[0].date)
+
+			res.render('page/course/module/events/attendee', {booking, eventDateWithMonthAsText})
+		}
+	}
+
+	public updateBooking() {
+		return async (req: Request, res: Response) => {
+			const bookings = res.locals.bookings
+			const bookingId = req.params.bookingId
+			const booking = bookings.find(function(booking: Booking) {
+				return booking.id == bookingId
+			})
+
+			if (req.body.type == 'unregister') {
+				booking.status = Booking.Status.REQUESTED
```

Comment:
I think we want to change the status to 'Cancelled' here. (It's not available in master yet though). I'm not sure we ever want to change a booking from 'Registered' to 'Requested'.

Reply:
I think cancelling will be done in a separate ticket, this was here because of an old design where you could confirm and unconfirm a booking

Reply:
I think perhaps we should discuss with Abi/Yuksel. I don't see the benefit of changing a booking from 'Confirmed' to 'Requested' and the learner record will fail validation on a request that attempts it so something needs to change.

Reply:
It's been noted on the ticket not to include this I think, I just missed it when reading the ticket, so I think we're fine to just remove it